templates/web.letsencrypt.ssl.template.yml

```diff
@@ -125,6 +125,10 @@ hooks:
        to: |
          ssl_certificate_key /shared/ssl/$$ENV_DISCOURSE_HOSTNAME.key;
          ssl_certificate_key /shared/ssl/$$ENV_DISCOURSE_HOSTNAME_ecc.key;
+         ssl_stapling on;
+         ssl_stapling_verify on;
```

Comment:
According to Nginx's doc,

> For verification to work, the certificate of the server certificate issuer, the root certificate, and all intermediate certificates should be configured as trusted using the [ssl_trusted_certificate](https://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_trusted_certificate) directive.

Are we missing the `ssl_trusted_certificate` directive for this to work?

Reply:
It's not required.